momkhulu/settings/base.py

```diff
@@ -38,10 +38,13 @@
 
 ROOT_URLCONF = "momkhulu.urls"
 
+LOGIN_REDIRECT_URL = "/"
+LOGOUT_REDIRECT_URL = "/"
```

Comment:
Sorry, a bit late to the party, but it's worth setting `LOGIN_URL` as well. That way, we can redirect to the login page when they are not logged in.
[Docs](https://docs.djangoproject.com/en/2.1/ref/settings/#login-url)

Reply:
This way, you don't have to keep on setting the login_url when using `@login_required()`